test/test_credit_details.rb

```diff
@@ -7,16 +7,9 @@ def setup
   end
 
   def test_get
-    response = client.credit_details.get(access_token)
-    refute_empty(response.accounts)
-    refute_empty(response.credit_details)
-
-    account_id = response.accounts[3].account_id
-    response = client.credit_details.get(
-      access_token,
-      account_ids: [account_id]
-    )
-    assert_equal(1, response.credit_details.size)
+    assert_raises(Plaid::InvalidRequestError) do
+      client.credit_details.get(access_token)
```

Comment:
Why should this error out?

Reply:
I would rm the test completely. These were just finally turned off. They've been deprecated for a while.

Reply:
I agree with removing them